Configurations.md

```diff
@@ -803,6 +803,140 @@ By default this option is set as a percentage of [`max_width`](#max_width) provi
 
 See also [`max_width`](#max_width) and [`width_heuristics`](#width_heuristics)
 
+## `fn_generics_space`
+
+Leave spaces around function generics.
+
+- **Default value**: `"Never"`
+- **Possible values**: `"OnlyBefore"`, `"OnlyAfter"`, `"BeforeAndAfter"`, `"Never"`
+- **Stable**: No (tracking issue: [#3564](https://github.com/rust-lang/rustfmt/issues/3564))
```

Comment:
Could you remove the link from this one (it'll help ensure we don't forget  to create the actual tracking issue :smile:)

 ```suggestion
- **Stable**: No
```

Reply:
Fixed. I thought tracking issues started from the original issue in this repository